spec/services/find_ancestors_service_spec.rb

```diff
@@ -0,0 +1,63 @@
+describe FindAncestorsService do
+  subject(:find_ancestors) { described_class.new(goods_nomenclature_item_id).call }
```

Comment:
```suggestion
  subject(:service) { described_class.new(goods_nomenclature_item_id).call }
```
See
```
spec/services/applicable_vat_options_service_spec.rb|2 col 3| subject(:service) { described_class.new(measures) }
spec/services/cached_geographical_area_service_spec.rb|2 col 3| subject(:service) { described_class.new(actual_date, countries) }
spec/services/cached_commodity_service_spec.rb|2 col 3| subject(:service) { described_class.new(commodity.reload, actual_date, filter_params) }
spec/services/quota_search_service_spec.rb|2 col 3| subject(:service) { described_class.new(filter, current_page, per_page) }
spec/services/applicable_additional_code_service_spec.rb|2 col 3| subject(:service) { described_class.new(measures) }
spec/services/exchange_rate_service_spec.rb|2 col 3| subject(:service) { described_class.new }
spec/services/measure_unit_service_spec.rb|2 col 3| subject(:service) { described_class.new(measures) }
```

Reply:
ok I'll change it,
but what do you think it's more readable
`find_ancestor` 
or
`service`?
😺

Reply:
When it comes to the subject under test I normally just look at the subject description (class name) at the top of the file. It's a bit like using `described_class`  the fully qualified name versus `FindAncestors`. I guess the service pattern is so common it makes sense to just follow a standard that codes to the interface and not to this specific implementation.